plugins/stingray/the_debuginator_plugin/c_api_the_debuginator.cpp

```diff
@@ -376,6 +383,35 @@ void handle_default_input(TheDebuginator* debuginator, unsigned devices) {
 					}
 				}
 
+	if (devices & Debuginator_Mouse) {
+		CApiInputControllerPtr mouse = script_api->Input->mouse();
+		input_wrapper.controller = mouse;
+
+		if (api->any_released(input_wrapper.controller) != UINT_MAX) {
+			input_wrapper.time_since_pressed = -1;
+		}
+
+
+		if (debuginator_is_open(debuginator)) {
+			CApiVector3 scroll = input_wrapper.axis("wheel");
+			if (scroll.y != 0) {
+				debuginator->focus_height += scroll.y * 0.05f;
```

Comment:
The usage request was "I want to be able to scroll, then if I don't click something and press a up I should go one up from the old selection, like a text-editor or file explorer would", which is why I'm doing this. It feels weird that the focus_height is outside [0,1] range and we're focusing the view around a hot item that's not on screen; probably rather want a view-offset and let the focus height be constant.